src/clearCompleted.js

```diff
@@ -0,0 +1,21 @@
+import { createList } from './addtask.js';
+
+export default function clearAll(items) {
+  const checkedTasks = [];
+  const deleteButton = document.querySelector('.delete');
+  deleteButton.addEventListener('click', () => {
+    for (let i = 0; i < items.length; i += 1) {
```

Comment:
It is also nice to avoid old for loop syntax, you could use a forEach instead, like the given example:
Instead of this:
`const list = [2, 5, 7, 2, 6, 2, 3, 5]
for (let i = 0; i < list.length; i++) {
    console.log(list[i])
}`

You could do this:
`list.forEach(number => console.log(number))`